spec/support/pages/admin_dashboard_page.rb

```diff
@@ -129,13 +129,16 @@ def username=(attr)
 
   def save
     click_on "Save"
-    wait_for_ajax
-    AdminDashboardPage.new(context: context)
+    AdminDashboardPage.new(context: context).tap do |page|
+      expect(page).not_to have_css('.overlay-container')
```

Comment:
might be good to just add this to the constructor for AdminDashboardPage.

Reply:
this method is actually on an edit modal overlay for the admin dashboard page. On calling the `save` or `cancel` methods it returns a new `AdminDashboardPage`. Adding this to the constructor would move this code away from the place where it is actually needed, when the user has taken an action to close the overlay.